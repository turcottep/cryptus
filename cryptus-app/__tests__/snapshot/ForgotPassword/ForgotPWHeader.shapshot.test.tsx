// components/__tests__/button.snapshot.test.tsx
import React from 'react';
import renderer from 'react-test-renderer';
import ForgotPWHeader from "../../../components/forgot_password/forgot_password_header"

it('renders correctly', () => {
  const title = "A title"
  const step = 2
  const tree = renderer.create(<ForgotPWHeader title={title} step={step}/>).toJSON()
  expect(tree).toMatchSnapshot()
})