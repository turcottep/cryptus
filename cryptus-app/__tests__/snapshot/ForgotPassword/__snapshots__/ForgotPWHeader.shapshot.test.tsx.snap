// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex flex-col mt-8"
>
  <div
    className="flex justify-center mx-auto"
  >
    <div
      className=""
    >
      <p
        className="text-brown text-3xl float-right mx-2 "
      >
        ●
      </p>
    </div>
    <div
      className=""
    >
      <p
        className="text-brown text-3xl float-left mx-2"
      >
        ●
      </p>
      <p
        className="text-gray-500 text-3xl float-left mx-2"
      >
        ●
      </p>
    </div>
  </div>
  <div
    className="flex flex-col mt-4"
  >
    <p
      className="text-black text-lg text-center"
    >
      A title
    </p>
  </div>
</div>
`;
